### PR TITLE
Format all files based on .editorconfig

### PR DESCRIPTION
**Pull request by loucass003**

### CODE REVIEW

File: src/main/java/dev/slimevr/bridge/BridgeThread.java [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-public @interface BridgeThread {
-	
-}
+package dev.slimevr.bridge;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(value = RetentionPolicy.SOURCE)
+public @interface BridgeThread {
```
Comment (TheDevMinerTV) [inline src/main/java/dev/slimevr/bridge/BridgeThread.java:7]:
```suggestion
public @interface BridgeThread {}
```

File: src/main/java/dev/slimevr/bridge/BridgeThread.java [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+
+}
```
Comment (TheDevMinerTV) [inline src/main/java/dev/slimevr/bridge/BridgeThread.java:9]:
```suggestion
```

File: src/main/java/dev/slimevr/bridge/BridgeThread.java [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-	
-}
+package dev.slimevr.bridge;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(value = RetentionPolicy.SOURCE)
+public @interface BridgeThread {
+
```
Comment (TheDevMinerTV) [inline src/main/java/dev/slimevr/bridge/BridgeThread.java:8]:
```suggestion
```

File: src/main/java/dev/slimevr/NetworkProtocol.java [hunk 4]
```diff
@@ -1,7 +1,7 @@
 package dev.slimevr;
 
 public enum NetworkProtocol {
-	
+
```
Comment (TheDevMinerTV) [inline src/main/java/dev/slimevr/NetworkProtocol.java:4]:
```suggestion
```

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

File: .editorconfig [hunk 5]
```diff
@@ -1,13 +1,1140 @@
...
-indent_style = tab
-end_of_line = lf
-charset = utf-8
-# This line causes problems with VSCode and potentially with other editors where all purely
-# whitespace lines are trimmed to be empty when saved, causing excessive worthless changes with Git
-#trim_trailing_whitespace = true
-insert_final_newline = true
+[*]
+charset = utf-8
+end_of_line = crlf
```
Comment (TheDevMinerTV) [inline .editorconfig:3]:
```suggestion
end_of_line = lf
```

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]
Please apply this and reformat <3

**Review by Eirenliel — APPROVED** [review]

**Pull request merged**